tests/Form/InputTextTest.php

```
@@ -0,0 +1,620 @@
+
+    public function testRenderWithValue(): void
+    {
+        self::assertSame(
+            <<<HTML
+            <input id="inputtext-" type="text" value="test">
+            HTML,
+            InputText::tag()->id('inputtext-')->value('test')->render(),
+            "Failed asserting that element renders correctly with 'value' attribute.",
+        );
+    }
+}
```

PHPMD (2.15.0)

[warning] The class InputTextTest has 52 public methods and attributes. Consider reducing the number of public items to less than 45. (undefined)

(ExcessivePublicCount)